airflow/www/fab_security/manager.py

```diff
@@ -208,12 +206,6 @@ def oauth_tokengetter(token=None):
     userstatschartview = UserStatsChartView
     permissionmodelview = PermissionModelView
 
-    @cached_property
-    def resourcemodelview(self):
```

Comment:
Just for my own understanding, this is being moved to the fab auth manager (in the security_manager_override). But all the code above is described as an Override, which to me overrides some default implementation. But here it looks like the default is being removed. So now I'm a little confused what other auth managers will do. Will it become actually required to override the security manager? Since there is no impl left here? Or with other auth managers use the fab security manager override class?

Reply:
Good question. The main goal is to, iteratively, move everything related to FAB from the different security managers (`airflow/www/fab_security/manager.BaseSecurityManager`, `airflow/www/fab_security/sqla/manager.SecurityManager`, `airflow/www/security.py.AirflowSecurityManager`) to the `FabAirflowSecurityManagerOverride`. Ideally, auth managers do not have to override the security manager, only if they want/need. The end goal is that everything we move from the security managers to `FabAirflowSecurityManagerOverride` are not needed if you dont use the FAB auth manager.

Reply:
I see, so the override class will be the new final home for all this code, and other auth managers don't need any of it (it's still not clear to me what they'll replace it with or why they don't need it, but I guess I don't _need_ those details).

My only last comment would be that calling that new class an Override seems a bit odd. It's not really an override if it's going to be the official/final location for all this code.

Reply:
It is still an override because *I think* Airflow needs a security manager regardless of the auth manager. I assume there is code in these security managers which will stay in the security manager. I am not sure what code and I guess this is something we'll find out while working on the AIP. But yes, I still expect to have a security manager by default which does the very minimal and nothing related to FAB auth. On top of that default security manager, auth managers (a lot of managers ...) can override it if they want/need. That's my current mental picture

Reply:
> I think Airflow needs a security manager regardless of the auth manager. I assume there is code in these security managers which will stay in the security manager.

I think this is the bit I've been trying to get at, the default implementation I mentioned above. Knowing that it makes a bit more sense. I'm curious to see what's left over after all the FAB stuff is removed, and to what degree any default implementation is left over.